overlord/snapstate/snapmgr.go

```diff
@@ -385,19 +385,13 @@ func (m *SnapManager) undoLinkSnap(t *state.Task, _ *tomb.Tomb) error {
 // be changed so it looks first at the state for the snap details (Revision, Developer, etc),
 // and then complements it with information from the snap itself.
 func SnapInfo(state *state.State, snapName string, snapRevision int) (*snap.Info, error) {
-	fname := filepath.Join(dirs.SnapSnapsDir, snapName, strconv.Itoa(snapRevision), "meta", "snap.yaml")
-	yamlData, err := ioutil.ReadFile(fname)
+	fname := filepath.Join(dirs.SnapSnapsDir, snapName,
+		strconv.Itoa(snapRevision), "meta", "snap.yaml")
```

Comment:
That would look nicer in a single line. Line will be shorted if parameter names are less redundant:

* snapName => name
* snapRevision => revision